docs/UserGuide.md

```diff
@@ -26,7 +26,7 @@ AddressBook Level 3 (AB3) is a **desktop app for managing contacts, optimized fo
 
    * **`list`** : Lists all contacts.
 
-   * **`add`**`n/John Doe p/98765432 e/johnd@example.com a/John street, block 123, #01-01` : Adds a contact named `John Doe` to the Address Book.
+   * **`addPatient`**`n/John Doe t/37.4 d/20200910-20200924 p/98765432 a/35` : Adds a quarantined individual named `John Doe` to the application.
```

Comment:
is it better if we do ```addpatient``` instead of ```addPatient```? just asking out of curiousity, no need to change if you think its fine

Reply:
Yep I will change to add patient instead!